docs/abi-spec.rst

```diff
@@ -608,6 +608,7 @@ Through ``abi.encodePacked()``, Solidity supports a non-standard packed mode whe
 
 - types shorter than 32 bytes are neither zero padded nor sign extended and
 - dynamic types are encoded in-place and without the length.
+- array and struct elements are padded, but still encoded in-place
```

Comment:
Didn't we say it won't support structs?

Reply:
As far as I can remember, we said that structs are unsupported for `abi.encodePacked`, but they are supported as indexed event  parameters - please correct me if I'm wrong.

Reply:
Yep, but we also said the documentation should distinguish between the two. This here is clearly about `abi.encodePacked`.